### DISCUSSION TITLE
Installing DinosOffice from GetIt in Delphi 12.1 did not work

### DISCUSSION THREAD

**Discussion by PeterPanino** [Daniel09Fernandes/Component_OpenOffice#61] [Announcements]
When trying to install DinosOffice from GetIt in Delphi 12.1 (Windows 11), I got this error message:

![image](https://github.com/user-attachments/assets/5cd88b20-e35c-47fb-a39a-4d6b91c32b02)

**Comment by Daniel09Fernandes** [comment]
I tesed here, and in my tested is OK.

If, in getit, marked how instaled, you can unistall. 

Go to your documents/Embarcadero/Studio/23.0/catalogRepository/ and delete the folderDinosOfficeLibreOffice

Install the LibreOffice https://pt-br.libreoffice.org/donate/dl/win-x86_64/24.2.5/pt-BR/LibreOffice_24.2.5_Win_x86-64.ms

Turn back to GetIt, and try install again.
